tests/test_gc.py

```
@@ -0,0 +1,390 @@
+    def test_no_match_random_branch(self) -> None:
+        assert extract_issue_from_branch("feature/login-page") == (None, None)
+
+    def test_chore_branch(self) -> None:
+        assert extract_issue_from_branch("chore/issue-5-deps") == ("5", None)
+
+    def test_docs_jira(self) -> None:
+        assert extract_issue_from_branch("docs/ABC-99-readme") == (None, "ABC-99")
+
+
+class TestCleanupByIssueState:
+    @pytest.fixture()
```

Ruff (0.15.21)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)